app/service/ingest.py

```diff
@@ -58,15 +60,17 @@ def save_collections(
 
 @validate_call(config=ConfigDict(arbitrary_types_allowed=True))
 def save_families(
-    family_data: list[dict], corpus_import_id: str, db: Optional[Session] = None
+    family_data: list[dict[str, Any]],
+    corpus_import_id: str,
+    db: Optional[Session] = None,
 ) -> list[str]:
     """
     Creates new families with the values passed.
 
-    :param list[dict] families_data: The data to use for creating families.
+    :param list[dict[str, Any]] families_data: The data to use for creating families.
     :param str corpus_import_id: The import_id of the corpus the families belong to.
-    :param Optional[Session] The database session to use for saving families.
-    :return str: The new import_ids for the saved families.
+    :param Optional[Session] The database session to use for saving families or None.
```

Comment:
Missing parameter name.
```suggestion
    :param Optional[Session] db: The database session to use for saving families or None.
```